uqcsbot/basic.py

```diff
@@ -1,18 +1,21 @@
-from discord.ext import commands
-import discord
 from typing import List
 
+import discord
+from discord import app_commands
+from discord.ext import commands
+
 UQCS_REPO_URL = "https://github.com/UQComputingSociety/"
 
 REPOS = {
-    "coc": ("code-of-conduct", "The UQCS Code of Conduct to be followed by all community members"),
+    "cpg": ("cpg", "Resources for the UQCS competitive programming group"),
+    "conduct": ("code-of-conduct", "The UQCS Code of Conduct to be followed by all community members"),
     "constitution": ("constitution", "All the business details"),
+    "cookbook": ("cookbook", "A cookbook of recipes contributed by UQCS members"),
     "design": ("design", "All UQCS design assets"),
     "events": ("events", "A repository for events and talk materials"),
-    "inviter": ("slack-invite-automation", "Web application to invite users to our slack team"),
     "minutes": ("minutes", "Minutes from UQCS committee meetings and general meetings"),
```

Comment:
This should probably be changed to "Minutes from UQCS general meetings", since we don't have committee minutes on there (I've updated the repo description as well).